src/components/repository/RepositoryTable.jsx

```diff
@@ -23,7 +23,10 @@ import { techNamesToDisplay } from 'utils/upload/fileUploadSpecifications';
 const { Paragraph } = Typography;
 
 const RepositoryTable = (props) => {
+  const [experimentCloning, setExperimentCloning] = useState(false);
```

Comment:
Just tested. I think the disable on every button makes sense but the spinners in all of the buttons look a bit weird.

What do you think about changing this to `useState(null);`

The we could use this `cloningExperimentId` or sth to show the loader in the button for that experiment (but disabling for every button if `cloningExperimentId` is not `null`)

Reply:
i'll change to `disabled`